core/src/main/java/org/mapfish/print/map/geotools/grid/RotationQuadrant.java

```diff
@@ -18,21 +18,21 @@
     QUADRANT_1 {
         @Override
         void updateTransform(final AffineTransform baseTransform, final int indent, final GridLabel.Side side,
-                             final int charHeight, final Rectangle2D textBounds) {
+                             final int halfCharHeight, final Rectangle2D textBounds) {
```

Comment:
Because I was lost. `charHeight` was not containing the height of the characters, but only half of that. I was a bit lost with the computations done in there because of that.
So I've renamed it to better describe what it represents (if you need a comment to describe what a variable represents, it means it is badly named).